stream-chat-android-ui-components/src/main/res/layout/stream_ui_item_message_plain_text.xml

```diff
@@ -8,6 +8,16 @@
     android:layout_marginEnd="8dp"
     >
 
+    <View
```

Comment:
Seems you don't need that. You can use messageContainer for the same purpose, can't you?

Reply:
right, thanks!